lib/mutant/mutator/node/send.rb

```diff
@@ -117,6 +118,22 @@ def emit_receiver_selector_mutations
             .each(&method(:emit_selector))
         end
 
+        # Emit mutation for `#dig`
+        #
+        # - Mutates `foo.dig(a, b)` to `foo.fetch(a).dig(b)`
+        # - Mutates `foo.dig(a)` to `foo.fetch(a)`
+        #
+        # @return [undefined]
+        def emit_dig_mutation
+          return unless selector.equal?(:dig) && arguments.any?
+
+          fetch_mutation = s(:send, receiver, :fetch, arguments.first)
```

Comment:
Heh I actually had exactly this code with different naming. The flay score was a bit higher and I didn't feel like it bought me much. I'm currently ambivalent so I'd be interested in hearing an argument for why this code is better.